lib/std/os/linux/tls.zig

```diff
@@ -539,7 +539,7 @@ inline fn mmap(address: ?[*]u8, length: usize, prot: usize, flags: linux.MAP, fd
             prot,
             @as(u32, @bitCast(flags)),
             @as(usize, @bitCast(@as(isize, fd))),
-            @as(usize, @truncate(@as(u64, @bitCast(offset)) / linux.MMAP2_UNIT)),
+            @as(usize, @truncate(@as(u64, @bitCast(offset)) / (linux.MMAP2_UNIT orelse std.heap.pageSize()))),
```

Comment:
Let's try just completely deleting `MMAP2_UNIT` and always using `std.heap.pageSize()` here.